docs/remote_inference_blueprints/open_ai_connector_chat_blueprint.md

```diff
@@ -0,0 +1,92 @@
+### OpenAI connector blueprint example for chat:
```

Comment:
Explain that this example is from OpenAI doc https://platform.openai.com/docs/api-reference/chat?

Reply:
Same for other examples, we should use the exact same example from OpenAI and Cohere official document. And we should put their doc link here so community user can refer to .